slack_rules/slack_app_access_expanded.yml

```diff
@@ -0,0 +1,208 @@
+AnalysisType: rule
+Filename: slack_app_access_expanded.py
+RuleID: Slack.AuditLogs.AppAccessExpanded
+DisplayName: Slack App Access Expanded
+Enabled: true
+LogTypes:
+  - Slack.AuditLogs
+Tags:
+  - Slack
+Severity: Medium
+Description: Detects when a Slack App has had its permission scopes expanded
+Reference: https://api.slack.com/admins/audit-logs
+DedupPeriodMinutes: 60
+Threshold:  1
+SummaryAttributes:
+  - action
+  - p_any_ip_addresses
+  - p_any_emails
+Tests:
+  -
+    Name: App Scopes Expanded
```

Comment:
I would like to see at least one of these test cases include the scope-related attributes for the scope-changed types of logs.